modules/openapi-generator/src/main/resources/typescript/http/http.mustache

```diff
@@ -287,6 +287,26 @@ export class ResponseContext {
         {{/node}}
         {{/platforms}}
     }
+
+    /**
+     * Use a heuristic to get a body of unknown data structure.
+     * Try JSON with fall back to plain text and binary.
+     */
+    public async getBodyAsAny(): Promise<any> {
+        try {
+            const body = await this.body.text();
+            try {
+                return JSON.parse(body);
```

Comment:
I suggest not to parse the json here, but simply pass it as a string to the exception

Reply:
In my use cases data is predominantly sent as json. So I thought this might make things easier. If it cannot be parsed as json the heuristic would still fall back to plain string.
What is the reason you would discourage json parsing, @macjohnny ?
What about only doing it when there is a suitable Content-Type header set?

Reply:
How do you want to go ahead with this @macjohnny .
I would love to get this moving.
If you feel strongly about removing the JSON parsing, I‘ll do the change.

Reply:
Sorry, forgot to reply.
Since it is for debugging purposes, we should avoid heuristic parsing, and simply log the string payload

Reply:
Ok, that makes sens. I removed the heuristic.